sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -183,7 +183,13 @@
 
   String CORE_FORCE_AUTHENTICATION_PROPERTY = "sonar.forceAuthentication";
   boolean CORE_FORCE_AUTHENTICATION_DEFAULT_VALUE = false;
+
+  /**
+   * @deprecated since 6.3. This feature is not supported anymore. See https://jira.sonarsource.com/browse/SONAR-7762
```

Comment:
you're almost right, I should have used @see => http://stackoverflow.com/questions/1082050/linking-to-an-external-url-in-javadoc